pkg/feature/raw_resources.go

```diff
@@ -40,20 +41,31 @@ func createResources(cli client.Client, objects []*unstructured.Unstructured, me
 
 		name := object.GetName()
 		namespace := object.GetNamespace()
+		managed, exists := object.GetAnnotations()[annotations.ManagedByODHOperator]
 
 		err := cli.Get(context.TODO(), k8stypes.NamespacedName{Name: name, Namespace: namespace}, object.DeepCopy())
-		if err == nil {
-			// object already exists, skip reconcile allowing users to tweak it
-			continue
-		}
-		if !k8serrors.IsNotFound(err) {
+		if err != nil && !k8serrors.IsNotFound(err) {
 			return errors.WithStack(err)
 		}
```

Comment:
done using `fmt.Errorf` since linter was complaining as  `errors.Errorf`  

"printf: github.com/pkg/errors.Errorf does not support error-wrapping directive %w (govet)"

Reply:
Bummer. I assumed it does...